### PR TITLE
Pm 2021 07 comment fixes

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by phillimorland** [comment]
this got a little out of hand, once again using px spacings due to it already being standard for many things :(

File: home/templates/home/blocks/tile_column_block.html [hunk 1]
```diff
@@ -1,110 +1,220 @@
...
+      </div>
+      {% endif %}
+    </div>
 
-          {% if self.image_col1 %}
-            {% image self.image_col1 fill-300x268 class="card-img-top" %}
-          {% else %}
-            <img src="{% static 'images/yellow-bg.png'%}" class="card-img-top" />
-          {% endif %}
+<!-- Mobile version -->
```
Comment (phillimorland) [inline home/templates/home/blocks/tile_column_block.html:17]: @fuzzylogic2000 this is the looping issue

File: home/templates/home/blocks/tile_column_block.html [hunk 2]
```diff
@@ -1,110 +1,220 @@
...
-          {% if self.image_col2 %}
-            {% image self.image_col2 fill-300x268 class="card-img-top" %}
-          {% else %}
-            <img src="{% static 'images/light-blue-bg.png'%}" class="card-img-top" />
+<!-- Desktop version -->
+      <div class="row d-none d-lg-flex">
+        <div class="col-lg-3 d-flex align-items-stretch">
+          <div class="card tile-block__tile">
+          {% if self.internal_link_col1 %}
+            <a href="{{ self.internal_link_col1.url }}">
```
Comment (phillimorland) [inline home/templates/home/blocks/tile_column_block.html:118]: @fuzzylogic2000 maybe here is better, all the cols are number to ensure notmore than 4 are added but becasue of that I couldn't figure out how to loop through, instead i'm duplicating code whish I don't like

**Review by fuzzylogic2000 — APPROVED** [review]
Cool, it looks good! 
And I am fine with leaving the columns like that. If we ever need to they will be easy to migrate.

**Pull request merged**